### PR TITLE
Removed unused import and updated the path.

### LINKED ISSUE #10835: update path handling in examples/preprocessing/otp.py
all of our `data_path` functions now return `pathlib.Path` objects, which allows things like the following:

https://github.com/mne-tools/mne-python/blob/37a9e5587443109515a85cdc4396af4e014ecf82/examples/datasets/brainstorm_data.py#L32-L35

Compare this to the old pattern (using `os.path.join`, often imported as `op.join`):

```py
raw_path = os.path.join(data_path, 'MEG', 'bst_raw',
                        'subj001_somatosensory_20111109_01_AUX-f.ds')
```

or the *really* old pattern (using the `+` operator on plain strings):

```py
raw_path = (data_path + '/MEG/bst_raw/' +
            'subj001_somatosensory_20111109_01_AUX-f.ds')
```

Update the paths in the file mentioned in the issue title to use the new pathlib-style, which is considered the modern, best-practice way to handle file paths.

### PR DESCRIPTION
**Pull request by architsinghal-mriirs**
Thanks for contributing a pull request! Please make sure you have read the
[contribution guidelines](https://mne.tools/dev/install/contributing.html)
before submitting.

Please be aware that we are a loose team of volunteers so patience is
necessary. Assistance handling other issues is very welcome. We value
all user contributions, no matter how minor they are. If we are slow to
review, either the pull request needs some benchmarking, tinkering,
convincing, etc. or more likely the reviewers are simply busy. In either
case, we ask for your understanding during the review process.

Again, thanks for contributing!

#### Reference issue
closes #10835

#### What does this implement/fix?
Explain your changes.
Removed Unused Import 'op'

#### Additional information
Any additional information you think is important.
The parenthesis is required for the 'read_raw_fif' import.

### CODE REVIEW

File: examples/preprocessing/otp.py [hunk 1]
```diff
@@ -35,7 +34,7 @@
 dipole_number = 1
 data_path = bst_phantom_elekta.data_path()
 raw = read_raw_fif(
-    op.join(data_path, 'kojak_all_200nAm_pp_no_chpi_no_ms_raw.fif'))
+    (data_path / 'kojak_all_200nAm_pp_no_chpi_no_ms_raw.fif'))
```
Comment (agramfort) [inline examples/preprocessing/otp.py:37]:
```suggestion
    data_path / 'kojak_all_200nAm_pp_no_chpi_no_ms_raw.fif')
```

**Comment by architsinghal-mriirs** [comment]
Hi @drammock I've made the changes in the same PR here.

**Comment by drammock** [comment]
@architsinghal-mriirs it seems you have a rebase problem, there are commits / changes in this PR that shouldn't be there (the changes to `test_search_light`).  Assuming that you've called the mne-tools repository "upstream", can you do

```
git switch rebase/updatepath  # if you're not already on that branch
git fetch upstream
git rebase upstream/main
git push -f
```

That should hopefully clean things up here.

**Comment by architsinghal-mriirs** [comment]
I apologize for the trouble @drammock , I'm still a student and learning. Could you please check if this is fixed now ?

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
thx @architsinghal-mriirs

**Pull request merged**